CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 ### Unreleased
 
+Improvements:
+- Use fuzzy matching for completion (thanks [Po Chen](https://github.com/princemaple)) [#491](https://github.com/elixir-lsp/elixir-ls/pull/491)
```

Comment:
function completion only, for now

Reply:
Ah, yes. Good point. Adjusted the wording in https://github.com/elixir-lsp/elixir-ls/pull/499/commits/4cdbd248d8f35f1649f14579da30c269a58f51cf